### PR TITLE
Update Deployments as code docs

### PR DESCRIPTION
**Pull request by jwitz**
Some small edits to our Deployments as code docs. We already have notes about the lack of support for programmatic template Deployment creation and connections, so just moved those around to be more intuitive. Also added back the metadata section in the example template file with the same comment as before.

### CODE REVIEW

File: astro/manage-deployments-as-code.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ description: "Manage an Astro Deployment with a JSON or YAML file."
...
 - Store a Deployment file that represents the configurations of an existing Deployment. You can make changes to this file to update a Deployment faster and more easily than doing so with the Cloud UI or individual Astro CLI commands.
 - Use a template file from an existing Deployment to create another Deployment with the same configurations. This is an alternative to creating a new Deployment in the Cloud UI and manually copying configurations.
 
-Astro supports updating a Deployment programmatically with [Deployment API keys](api-keys.md), but you cannot currently automate creating a Deployment as part of a CI/CD workflow. You also cannot programmatically update or copy Airflow connections and variables across Deployments. Support for both of these features is coming soon.
+Astro supports updating a Deployment programmatically with [Deployment API keys](api-keys.md), but you cannot currently automate creating a Deployment as part of a CI/CD workflow. Support for both of these features is coming soon.
```
Comment (kwcanuck) [inline astro/manage-deployments-as-code.md:17]:
```suggestion
Astro supports updating a Deployment programmatically with [Deployment API keys](api-keys.md), but you can't  automate creating a Deployment as part of a CI/CD workflow.
```
We should try and avoid future looking statements. They can be a maintenance nightmare and could potentially cause embarrassment for the company.

File: astro/manage-deployments-as-code.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The output of this command includes the name, description, and metadata that is
 
 ### Template file reference
 
-When you inspect a Deployment and create a template file, the file will contain the following sections:
+When you inspect a Deployment, aa template file is created with the following sections:
```
Comment (kwcanuck) [inline astro/manage-deployments-as-code.md:50]:
```suggestion
When you inspect a Deployment, a template file is created with the following sections:
```

File: astro/manage-deployments-as-code.md [hunk 3]
```diff
@@ -90,9 +90,22 @@ deployment:
...
+        airflow_version: 
+        status: 
+        created_at: 
+        updated_at: 
+        deployment_url: 
+        webserver_url: 
 ```
 
-The following sections in this guide include information about each key and value within these sections in the template file.
+Read the following sections to learn about each available key and value in the template file.
```
Comment (kwcanuck) [inline astro/manage-deployments-as-code.md:108]:
```suggestion

```
This should be obvious.  If not, link to the related sections, or add a _see_ statement after each entry.

File: astro/manage-deployments-as-code.md [hunk 4]
```diff
@@ -142,6 +155,8 @@ To create a new Deployment from an existing template file:
    
    You can also go to the Workspace page in the Cloud UI.
 
+4. Optional. Reconfigure any Airflow connections or variables from the Deployment which you copied into the template file. Airflow connections and variables cannot be configured using template files. See [Manage connections in Airflow](https://docs.astronomer.io/learn/connections).
```
Comment (kwcanuck) [inline astro/manage-deployments-as-code.md:158]:
```suggestion
4. Optional. Reconfigure any Airflow connections or variables from the Deployment that you copied into the template file. Airflow connections and variables cannot be configured using template files. See [Manage connections in Airflow](https://docs.astronomer.io/learn/connections).
```

**Review by kwcanuck — APPROVED** [review]

File: astro/manage-deployments-as-code.md (hunk 1 above)
Comment (jwitz) [reply astro/manage-deployments-as-code.md:17]: I agree wholeheartedly!

**Pull request merged**